src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -293,13 +175,11 @@ private void EventLoop()
         {
             try
             {
-                bool shutdown = false;
                 Interop.Sys.SocketEvent* buffer = _buffer;
                 ConcurrentDictionary<IntPtr, SocketAsyncContextWrapper> handleToContextMap = _handleToContextMap;
                 ConcurrentQueue<SocketIOEvent> eventQueue = _eventQueue;
-                IntPtr shutdownHandle = ShutdownHandle;
                 SocketAsyncContext? context = null;
```

Comment:
Move down to usage? i.e. just before 
`if (handleToContextMap.TryGetValue(handle, out SocketAsyncContextWrapper ...`

Reply:
@benaadams  I suggest to deal with these concerns in #37138. Feedback is welcome there!